app/brave_main_delegate.cc

```diff
@@ -195,6 +196,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
 #if defined(OS_WIN)
       features::kWinrtGeolocationImplementation.name,
 #endif
+      omnibox::kOmniboxContextMenuShowFullUrls.name,
```

Comment:
We need to enable this feature because
`ChromeLocationBarModelDelegate::ShouldPreventElision` will check it along with the pref